GPy/models/gp_multiout_regression_md.py

```diff
@@ -0,0 +1,173 @@
+# Copyright (c) 2017  Zhenwen Dai
+# Licensed under the BSD 3-clause license (see LICENSE.txt)
+
+import numpy as np
+from ..core import SparseGP
+from .. import likelihoods
+from .. import kern
+from .. import util
+from GPy.core.parameterization.variational import NormalPosterior, NormalPrior
+from ..core.parameterization.param import Param
+from paramz.transformations import Logexp
+from ..util.linalg import tdot
+from .sparse_gp_regression_md import SparseGPRegressionMD
+
+class GPMultioutRegressionMD(SparseGP):
```

Comment:
Needs adding to release notes for new deploy. Is there some description/notebooks showing each of these working in practice and how they differ?

Also I've been documenting all the docstrings, and notebooks, for the entirety of GPy (coming soon) but since I'm not familiar with these models and a couple more, I haven't been able to fill them out on your behalf (I've no idea what Z_row, qU_var_r_W_dim, etc mean).Could you fill them in if you get chance? I understand we're in a hurry for this deploy though so perhaps just add it to a todo list if you don't have time right now?

Reply:
Thanks so much for doing the documentation!

The new model is for the nips paper. I will write the documentation. I think it can get it done this weekend.

Reply:
Great thanks! I've so far done core, likelihood, mappings, most models (except your new NIPS ones and the spike and slab and IBP ones ones as I don't have time to read the papers to figure out the inputs). I'm going to work on kernels, inference and plotting and the make a huge PR.